packages/react/accordion/src/Accordion.tsx

```diff
@@ -301,9 +301,9 @@ const AccordionItem = React.forwardRef((props, forwardedRef) => {
     <AccordionItemProvider open={open} disabled={disabled} buttonId={buttonId}>
       <CollapsiblePrimitive.Root
         data-state={open ? 'open' : 'closed'}
-        disabled={disabled}
         {...accordionItemProps}
         ref={forwardedRef}
+        disabled={disabled}
```

Comment:
I think it might be more consistent if we stick to the way we do it everywhere else lie:

```tsx
disabled={context.disabled}
{...props}
```